spec/models/upload_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe Upload, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
[requirement]

If there are no model specs (and I don't think there needs to be at the moment) then this file should be removed. It can be added later when model specs are needed.